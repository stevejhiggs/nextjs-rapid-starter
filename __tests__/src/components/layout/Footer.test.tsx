import React from 'react';
import { cleanup, render } from '@testing-library/react';
import Footer from '@/components/layout/Footer';

afterEach(cleanup);

describe('Components -> Footer', () => {
  it('should match the snapshot', () => {
    const { container } = render(<Footer />);
    expect(container).toMatchSnapshot();
  });
});
