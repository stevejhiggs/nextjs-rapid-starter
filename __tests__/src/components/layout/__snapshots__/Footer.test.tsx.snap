// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components -> Footer should match the snapshot 1`] = `<div />`;
